main/src/kotlinx/team/infra/Publishing.kt

```diff
@@ -91,11 +91,14 @@ internal fun Project.configurePublishing(publishing: PublishingConfiguration) {
         if (verifySonatypeConfiguration()) {
             includeProjects.forEach { subproject ->
                 subproject.createSonatypeRepository()
-                subproject.configureSigning()
             }
         }
     }
 
+    includeProjects.forEach { subproject ->
+        subproject.configureSigning()
```

Comment:
This may result in additional annoying warning in local builds where signing credentials are not provided.

Reply:
Now the signing is attempted only when running on TeamCity